_layouts/single.html

```diff
@@ -0,0 +1,8 @@
+---
+layout: base
+---
+<div class="container">
+  <article class="section content">
```

Comment:
This still gets very wide on regular screens, resulting in line widths that are very hard to read: https://baymard.com/blog/line-length-readability

I'd make this similar to `page.html`, but without the `<aside>`. And possibly rename this to `page-no-toc.html` or something like that, since "single" doesn't tell me a whole lot either.

Reply:
Are you suggesting keeping it at two-thirds width minus the 1/3 that was used for the side-nav?

Reply:
Yep!

Reply:
That column-based mechanism from Bulma makes that resizing pretty awkward. I'm thinking this is probably something we need to fix with the content itself. These pages affected by it still need cleanup.